mantidproject/mantid
# Convert Group 14 from Python 2 to Python 3

This PR deals with the Python2 to Python3 conversion of things in `scripts/HFIR_4Circle_Reduction/` towards #18550.

**To test:**

Code review
Go to Interfaces -> Diffraction -> HFIR 4Circle Reduction
Confirm that there are no import or general errors when opening the interfaces
Play around with the interfaces (no data needed)
Confirm that there are no Python related errors
Does not need to be in the release notes.
Description of work.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
